.github/workflows/sync-malicious-packages.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+  
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  synchronize-malicious-packages:
+    name: 'Synchronize malicious packages'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12' 
+
+      - name: Install AWS CLI
+        run: |
+          curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip
+          unzip awscliv2.zip
+          sudo ./aws/install
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-3
+          role-to-assume: ${{ secrets.IAM_ROLE_ARN }}
+          role-session-name: github-actions
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r scripts/sync-malicious-packages/requirements.txt
+
+      - name: Synchronize malicious packages
+        run: |
+          python scripts/sync-malicious-packages/sync-malicious-packages.py --since 2024-01-01 --destination ./samples/pypi --s3-bucket ${{ secrets.S3_BUCKET_NAME }} --dynamodb-table ${{ secrets.DYNAMODB_TABLE_NAME }} --ecosystem pypi
```

Comment:
I assume we're just testing with pypi and npm will be added later

Reply:
yes!